--- a/lib/backup/manager.rb
+++ b/lib/backup/manager.rb
@@ -95,7 +95,10 @@
         puts "  version: #{settings[:gitlab_version]}".red
         puts
         puts "Hint: git checkout v#{settings[:gitlab_version]}"
-        exit 1
+#        exit 1
+        puts "This check does not work with the packaged version!".red
+        puts "The dummy repository in /usr/share/gitlab gives only".red
+        puts "a randomized commit-hash so we continue with restore!".red
       end
     end
 
